### PR TITLE
Update notes_controller_test.rb

### PR DESCRIPTION
**Pull request by Yavnikaa**
Fixes part of #10808

### CODE REVIEW

**Comment by Yavnikaa** [comment]
@TildaDares why is  `tests / functional-tests (pull_request) ` failing? What do I do to prevent it?

**Comment by TildaDares** [comment]
Hi @Yavnikaa, the tests are failing because the changes from #10784 haven’t been merged.

File: test/functional/notes_controller_test.rb [hunk 1]
```diff
@@ -444,7 +444,7 @@ def teardown
     assert_response :success
     assert_not_nil @response.body
     json = JSON.parse(@response.body)
-    assert_equal ["can't be blank", "is too short (minimum is 3 characters)", "can only include letters, numbers, and dashes"], json['title']
+    assert_equal ["can't be blank", "is too short (minimum is 2 characters)", "can only include letters, numbers, and dashes"], json['title']
```
Comment (TildaDares) [inline test/functional/notes_controller_test.rb:447]: The changes in #10809 and this one has to be in one PR since they resolve the same thing.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: test/functional/notes_controller_test.rb (hunk 1 above)
Comment (Yavnikaa) [reply test/functional/notes_controller_test.rb:447]: Alright, I'll do it!

**Comment by Yavnikaa** [comment]
I am closing this as I have done the required changes with the [PR of the other file](https://github.com/publiclab/plots2/pull/10809).

**Pull request closed without merging**